spark-indexer/src/main/java/org/uniprot/store/spark/indexer/taxonomy/mapper/TaxonomyStatisticsAggregationMapper.java

```diff
@@ -4,20 +4,25 @@
 import org.uniprot.core.taxonomy.TaxonomyStatistics;
 import org.uniprot.core.taxonomy.impl.TaxonomyStatisticsBuilder;
 import org.uniprot.store.spark.indexer.common.util.SparkUtils;
+import org.uniprot.store.spark.indexer.taxonomy.mapper.model.TaxonomyStatisticsWrapper;
 
 public class TaxonomyStatisticsAggregationMapper
-        implements Function2<TaxonomyStatistics, TaxonomyStatistics, TaxonomyStatistics> {
+        implements Function2<
+                TaxonomyStatisticsWrapper, TaxonomyStatisticsWrapper, TaxonomyStatisticsWrapper> {
 
     private static final long serialVersionUID = -2085055845000585882L;
 
     @Override
-    public TaxonomyStatistics call(TaxonomyStatistics stat1, TaxonomyStatistics stat2)
+    public TaxonomyStatisticsWrapper call(
+            TaxonomyStatisticsWrapper wrapper1, TaxonomyStatisticsWrapper wrapper2)
```

Comment:
are wrapper1 and 2 relating to the organism + proteome tax wrappers? If so, could we name these variables differently, to reflect this?